### PR TITLE
batch input modal

### PR DESCRIPTION
**Pull request by yneves**

### CODE REVIEW

**Comment by yneves** [comment]
Ready to review @ntgn81

File: src/components/documents/DocumentBatchModal.js [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+          onChange={::this.onChange}
+          spellCheck={false}
+          style={styleTextarea}
+        />
+      </Field>
+    );
+  }
+
+  processText() {
+    const regex = /([0-9]{7})|([0-9]{11})|([0-9]{1},?[0-9]{3},?[0-9]{3},?)/g;
```
Comment (ntgn81) [inline src/components/documents/DocumentBatchModal.js]: [0-9] can be replaced with `\d`

**Comment by ntgn81** [comment]
We don't have unit tests yet, right?

File: src/components/documents/DocumentsList.js [hunk 2]
```diff
@@ -70,6 +71,14 @@ export default class DocumentsList extends Component {
     return docs.map(doc => new DocumentRecord(doc));
   }
 
+  onNumberKeyDown(e, doc) {
+    // 186 = semi-colon
```
Comment (ntgn81) [inline src/components/documents/DocumentsList.js]: This is nice, I was JUST going to ask what 186 is lol

**Comment by ntgn81** [comment]
:+1:

**Comment by yneves** [comment]
I'll add the unit tests!

**Pull request merged**